example/ndpiReader.c

```diff
@@ -1110,6 +1124,9 @@ static void parseOptions(int argc, char **argv) {
       for(thread_id = 1; thread_id < num_threads; thread_id++)
         _pcap_file[thread_id] = _pcap_file[0];
     }
+
+    if(num_threads > 1)
```

Comment:
Shouldn't that be:

```suggestion
    if(num_threads == 1)
```

or did I miss smth?

Reply:
got it..

Reply:
Fixed. Thanks!